src/oge/data_cleaning.py

```diff
@@ -72,13 +72,11 @@ def clean_eia923(
             All three data frames are at the subplant level.
     """
     # Allocate fuel and generation across each generator-pm-energy source
-    gf = load_data.load_pudl_table(
-        "denorm_generation_fuel_combined_monthly_eia923", year
-    )
-    bf = load_data.load_pudl_table("denorm_boiler_fuel_monthly_eia923", year)
-    gen = load_data.load_pudl_table("denorm_generation_monthly_eia923", year)
-    gens = load_data.load_pudl_table("denorm_generators_eia", year)
-    bga = load_data.load_pudl_table("boiler_generator_assn_eia860", year)
+    gf = load_data.load_pudl_table("out_eia923__monthly_generation_fuel_combined", year)
+    bf = load_data.load_pudl_table("out_eia923__monthly_boiler_fuel", year)
+    gen = load_data.load_pudl_table("out_eia923__monthly_generation", year)
+    gens = load_data.load_pudl_table("_out_eia__yearly_generators", year)
```

Comment:
I guess you checked that it is correct but I ask because it looks weird, should `_out` be `out` ?

Reply:
I am just using the specific table used by pudl in their version of the pipeline here: https://github.com/catalyst-cooperative/pudl/blob/b096fbc63bd5cb5f28c7d6496c618b8a36aa46bd/src/pudl/analysis/allocate_gen_fuel.py#L223. 

I do not completely understand the difference between `_out_eia__yearly_generators` and `out_eia__yearly_generators`

Reply:
See: 
<img width="422" alt="image" src="https://github.com/singularity-energy/open-grid-emissions/assets/45949268/994637bf-366c-46ea-91c7-24c27ada1902">

Reply:
It looks like these tables (the intermediate starting with `_`) will be removed from the database soon. Do we know what to do when this happens?

Reply:
I can ask

Reply:
Looking at the differences between these two tables:
<img width="809" alt="image" src="https://github.com/singularity-energy/open-grid-emissions/assets/45949268/e7c7fbbc-8bdb-4f63-afd9-1640eed908d1">
It looks like they have the same number of rows, but the new table has 8 more columns than in the legacy table. I think based on this, it would be safe to switch to the new table.